main.go

```diff
@@ -55,6 +56,7 @@ func main() {
 
 	prometheus.MustRegister(collector.NewClusterHealth(logger, httpClient, esURL))
 	prometheus.MustRegister(collector.NewNodes(logger, httpClient, esURL, *esAllNodes))
+	prometheus.MustRegister(collector.NewIndices(logger, httpClient, esURL, *esAllNodes, *esExportIndices))
```

Comment:
please replace by 
```
if *exExportIndices {
    prometheus.MustRegister(collector.NewIndices(logger, httpClient, esURL, *esAllNodes))
}
```
(so remove the flag from the NewIndices constructor as well)